src/dateTimePicker.story.jsx

```diff
@@ -0,0 +1,55 @@
+import React from 'react';
+import DateTimePicker from './DateTimePicker';
+import { storiesOf } from '@kadira/storybook';
+
+storiesOf('DateTimePicker', module)
+	.add('default', () => {
```

Comment:
use `addWithInfo` (see `popover.story.jsx` for an example). The `InfoWrapper` component gives us a nice documentation add-on